Source/Core/DolphinQt/Settings/InterfacePane.cpp

```diff
@@ -264,5 +274,13 @@ void InterfacePane::OnSaveConfig()
         tr("You must restart Dolphin in order for the change to take effect."));
   }
 
+  const bool use_covers = m_checkbox_use_covers->isChecked();
+
+  if (use_covers != Config::Get(Config::MAIN_USE_GAME_COVERS))
+  {
+    Config::SetBase(Config::MAIN_USE_GAME_COVERS, use_covers);
+    Settings::Instance().RefreshGameList();
```

Comment:
This still does unnecessarily much work.

Reply:
How to avoid it though?

Reply:
The simplest way would be to have GameTracker's thread call GameListCache::UpdateAdditionalMetadata. If you want to get fancier than that, you could make a variant of UpdateAdditionalMetadata that only does cover downloading and not Wii/custom banner loading, but that's not necessary in my opinion.

Reply:
That seems to only update the meta data for a single entry though. I'm still not sure how to even get to the point where I have a ``GameFile`` to call ``UpdateAdditionalMetadata`` on.

Reply:
You're supposed to call this version of `UpdateAdditionalMetadata`, which goes through all files in the cache rather than taking a `GameFile` as a parameter: https://github.com/dolphin-emu/dolphin/blob/ea99cdd781318c0883f1824a032d137854555923/Source/Core/UICommon/GameFileCache.h#L42-L43